did_doc_sov/src/lib.rs

```diff
@@ -139,3 +154,87 @@ impl<'de> Deserialize<'de> for DidDocumentSov {
         })
     }
 }
+
+impl Serialize for DidDocumentSov {
```

Comment:
add tests for `serialization -> deserialization`, assure we don't duplicate services